src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--start-days-wed calendar--month-length-30">
+        <p class="calendar__days"></p>
```

Comment:
<p> here is not the best choice. I would chose just div, but okay, can be

Reply:
Okey